src/amo/components/EditableCollectionAddon/index.js

```diff
@@ -129,7 +129,7 @@ export class EditableCollectionAddonBase extends React.Component<
           <div className="EditableCollectionAddon-notes">
             <h4 className="EditableCollectionAddon-notes-header">
               <Icon name="comments-blue" />
-              {i18n.gettext('User comment')}
+              {i18n.gettext('Leave a note')}
```

Comment:
This should be "Leave a note" when in edit note mode for the note, i.e., when editing a note or creating a new note, but it should be "Collector's note" when in view mode for the note, i.e., when in edit mode for the collection, but not yet editing the individual note.

Reply:
Yes, that's correct. 😄